tests/eth2/beacon/test_helpers.py

```diff
@@ -708,8 +708,8 @@ def mock_get_randao_mix(state,
                             latest_randao_mixes_length):
         return hash_eth2(
             state.root +
-            abs(epoch).to_bytes(32, byteorder='big') +
-            latest_randao_mixes_length.to_bytes(32, byteorder='big')
+            abs(epoch).to_bytes(32, byteorder='little') +
```

Comment:
`abs()` could be removed now.
```suggestion
            epoch.to_bytes(32, byteorder='little') +
```